### ISSUE TITLE
Write GitHub action to post documentation to readthedocs.org for new releases

### ISSUE THREAD

**Issue by jrsmith3** [jrsmith3/ibei#75]
# Overview
The scope of this issue is to upload the documentation build by the automation described in #55 to readthedocs.org via GitHub action. Documentation should only be updated in this way for releases.

# Related issues
* Depends on #52.
* Depends on #55.

**Comment by jrsmith3** [comment]
Superseded by #79.
